src/support/README.md

```diff
@@ -1,8 +1,15 @@
-
 # Support and feedback
 
 Thank you for exploring NodeScript. Please note that we're currently in an active development phase, and your feedback is invaluable to us. You can contact us at: 
 
-hello@nodescript.dev
+[support@nodescript.dev](mailto:support@nodescript.dev)
 
 If you have any questions or need help with a specific issue, please check our Frequently Asked Questions (FAQ) section. We're committed to making NodeScript better every day and we appreciate your contribution to this journey.
+<script>
+    var checkSlaask = setInterval(function() {
+    if (window._slaask) {
+        window._slaask.show();
+        clearInterval(checkSlaask);
+    }
+}, 100); 
+</script>
```

Comment:
This script is for automatically opening the chat when opening the support page. However I am not sure about if this is a good idea 🤔  In some resolutions it can block the page text.